service-deployment/bootstrap/grafana/grafana-configuration/pai-nodeview-dashboard.json.template

```diff
@@ -194,6 +194,13 @@
               "legendFormat": "free",
               "refId": "E",
               "step": 600
+            },
+            {
+              "expr": "node_memory_MemTotal{instance=~'$node'}",
```

Comment:
>node_memory_MemTotal{instance=~'$node' [](start = 23, length = 38)

this is just a flat line that never changes. can you change the line to "cache"? or just use two lines: free mem, and (total_mem - free_mem)

Reply:
change to 2 line:
free mem, and (total_mem - free_mem)

Reply:
![image](https://user-images.githubusercontent.com/5576848/39221357-0acb10ec-4869-11e8-8fea-490884678a5c.png)